getBaseJestConfig.cjs

```diff
@@ -0,0 +1,60 @@
+/*!
```

Comment:
Does this need to live at the root? Can we just have this live in the now-private package and pull in via workspace dev dependency?

Reply:
Hmm that's a good point, as long as it's a `.cjs` file and doesn't require the private package to be built in order to be imported from it, then it shouldn't run into the problem that is discussed in another thread up here... I'll give it a shot.

Reply:
I suggest putting this in build-common if possible. That's where we put the rest of the shared tsconfigs.